mmuser.go

```diff
@@ -53,6 +54,9 @@ func (u *User) loginToMattermost() (*matterclient.MMClient, error) {
 	if u.Cfg.Insecure {
 		mc.Credentials.NoTLS = true
 	}
+	if !u.Cfg.SkipTLSVerify {
+		mc.Credentials.SkipTLSVerify = true
+	}
```

Comment:
Could you change those 3 lines to ```mc.Credentials.SkipTLSVerify = u.Cfg.SkipTLSVerify```  ?
(looks like I can do the same with my u.Cfg.Insecure lines above ;-) )